scripts/build/dev_generator.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package build
+
+import (
+	"os"
+)
+
+func init() {
+	buildGenerators["dev"] = devGenerator{}
+}
+
+type devGenerator struct {
+}
+
+// There are a few dev nodes which are required to run an initramfs.
+func (g devGenerator) generate(config Config) ([]file, error) {
```

Comment:
I simply copied these verbatim from "scripts/dev.cpio".

Is it enough to only have these three files: "dev/console", "dev/ttyS0" and "dev/null"?